src/main/java/net/datafaker/providers/base/Image.java

```diff
@@ -158,8 +158,7 @@ private String generateBase64RasterImage(ImageType imageType, int width, int hei
             byte[] imageBytes = baos.toByteArray();
             return "data:" + imageType.mimeType + ";base64," + Base64.getEncoder().encodeToString(imageBytes);
         } catch (IOException e) {
-            e.printStackTrace();
-            return null;
+            throw new RuntimeException("Failed to generate image %s of size %sx%s".formatted(imageType, width, height), e);
```

Comment:
I don't see why you'd want to throw an exception here.

Reply:
Because it's much better to clearly show the problem instead of hiding it.
* When you return null, it almost always causes NullPointerException in the next step. And user will not understand what happened. For example, in DF own test: 
```java
        assertThat(faker.image().base64TIFF()).startsWith("data:image/tiff;base64,");
```

* If user wants to protect, he will need to write IF:

```java
String image = faker.image().base64TIFF();
if (image == null) {
  // WHAT SHOULD I DO HERE?..
  throw new RuntimeException("WTF, the image is null, and I have no idea why.");
}
```

Reply:
How is writing that if(null) check different from:

```
try {
   String image = faker.image().base64TIFF();
} catch(Exception e) {
  throw new RuntimeException("WTF, the image generation failed");
}
```

Reply:
It differs a lot.
* User doesn't need to check if image is null. 
* User doesn't need to pollute his code with unneeded IFs and try/catches. 
* User will see a clear error message explaining WHY the image could not be generated. 

> The implicit contract in DF is that none of the methods will ever throw an exception

It sounds very strange to me. 
Nobody can ever guarantee that his code never throws any exceptions. 

Anyway, in this case, such an exception actually cannot happen. We need to catch `IOException` only because java interfaces like `InputStream` declare `throws IOException`. But in this case we operate with `ByteArrayOutputStream` which **never throws `IOException`**.